index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+
+    <meta charset="utf-8">
+    <title>About Me - Day Two</title>
+    <link type="text/css" rel="stylesheet" href="style.css"/>
+  </head>
+
+  <body>
+      <h1>About Me</h1>
+
+      <p>My name is <strong>Jeff Ford</strong> and I was born on April 12th of 1980, in Aberdeen, Washington. The youngest of three children in my family, I grew up in Aberdeen and attended the <a href="http://www.asd5.org/highschool">local public high school</a>, where I was a left-handed pitcher on the baseball team.</p>
+
+      <p>After high school, I moved to Seattle to attend the University of Washington.  I eventually graduated from UW with a Bachelor's degree in political science.</p>
+
+      <p>I began working in the state legislature after college, and soon joined the Army Reserves. The decision to join the Army Reserves was a big one, and it had a more significant impact on my life than I expected.  In my thirteen years of service, I was on active-duty for five of them.</p>
+
+      <p>One benefit of the military was that I had the opportunity to see new places.  Places that I have lived because of the military include:
+          <ul class="places"><em>
```

Comment:
this `<ul>` is a sibling of the `<p>` tag, so they should be indented to the same level